### PR TITLE
Integrate Atari Env, Fixed bugs related to enjoy_appo

### PR DESCRIPTION
**Pull request by Zhehui-Huang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,33 +17,38 @@ Please cite the paper if you use the code from this repository in your work.
...
+conda activate scalable_agent
+```
 
-[TensorFlow][tensorflow] >=1.9.0-dev20180530, the environment
-[DeepMind Lab][deepmind_lab] and the neural network library
-[DeepMind Sonnet][sonnet]. Although we use [DeepMind Lab][deepmind_lab] in this
-release, the agent has been successfully applied to other domains such as
-[Atari][arxiv], [Street View][learning_nav] and has been modified to
-[generate images][generate_images].
+- Download deepmind_lab-1.0-py3-none-any.whl from https://drive.google.com/file/d/10SjkK7AcegkWBgx31-fbKTNRgNLd0wax/view?usp=sharing
```
Comment (alex-petrenko) [inline README.md:29]: No need for the google drive link. Just put this file in the repository, its not that big, we can do it.

File: environments_doom.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 DOOM_W = 128
 DOOM_H = 72
-
+DOOM_DIMENSION = 3
```
Comment (alex-petrenko) [inline environments_doom.py]: Call it DOOM_CHANNELS

**Review by alex-petrenko — CHANGES_REQUESTED** [review]
Please address my comments and we are good to go.

**Comment by Zhehui-Huang** [comment]
For deepmind_lab-1.0-py3-none-any.whl, it is 481.4 MB, so, I would prefer
to provide a link.

**Comment by alex-petrenko** [comment]
The general idea is that if you make sure everything is in one place, it's better. You may forget about this link and delete the file or otherwise lose it, but the repo will still be here.

But it's not super important, so I'm not going to argue anymore. Link it is.

**Pull request merged**